### PR TITLE
Onwubiko-Chibuike

### PR DESCRIPTION
**Pull request by prince-curie**

### CODE REVIEW

**Comment by ashinzekene** [comment]
Your project doesn't work yet???

**Comment by prince-curie** [comment]
it is working what i am currently working on is the search functionality, i am able to get news

**Comment by prince-curie** [comment]
Sorry for sending the wrong files earlier, i have pushed in the right files.

**Comment by ashinzekene** [comment]
Welldone. I've  been seeing all your work

File: Onwubiko-Chibuike/main.js [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+    var data = new URLSearchParams();
+    for (let pair of formData) {
+        data.append(pair[0],pair[1]);
+    }
+
+    data.append('apikey', apikeys);
+    data.toString();
+
+    let searchUrl = 'https://newsapi.org/v2/top-headlines?' + data;
```
Comment (ashinzekene) [inline Onwubiko-Chibuike/main.js]:
```suggestion
    let searchUrl = 'https://newsapi.org/v2/everything?' + data;
```

Comment (prince-curie) [reply Onwubiko-Chibuike/main.js]: changed it and i am getting an error 405 anytime i try searching online

Comment (prince-curie) [reply Onwubiko-Chibuike/main.js]: Solved sir.

Comment (prince-curie) [reply Onwubiko-Chibuike/main.js]: done

Comment (prince-curie) [reply Onwubiko-Chibuike/main.js]: Done

**Pull request merged**